### PR TITLE
Search Feature for Component Props

### PR DESCRIPTION
**Pull request by ghana7989**
Adding Search Feature for component props. Which completes the requirement as in here https://trello.com/c/obo7dTPn

### CODE REVIEW

File: docs/src/components/MdxPage/MdxPageTabs/MdxPageTabs.tsx [hunk 1]
```diff
@@ -65,6 +66,17 @@ export function MdxPageTabs({ body, frontmatter, headings }: MdxPage) {
               style={{ maxWidth: 1080, margin: 'auto', marginTop: 24 }}
               className={classes.tabContent}
             >
+              <Input
```
Comment (rtivital) [inline docs/src/components/MdxPage/MdxPageTabs/MdxPageTabs.tsx:69]: Replace Input with TextInput, Input should only be used to build other inputs

File: docs/src/components/MdxPage/MdxPageTabs/MdxPageTabs.tsx [hunk 2]
```diff
@@ -65,6 +66,17 @@ export function MdxPageTabs({ body, frontmatter, headings }: MdxPage) {
...
+                variant="default"
+                placeholder="Search Props"
+                style={{
+                  marginBottom: 20,
+                }}
+                value={searchValue}
+                onChange={(event: { currentTarget: { value: React.SetStateAction<string> } }) =>
```
Comment (rtivital) [inline docs/src/components/MdxPage/MdxPageTabs/MdxPageTabs.tsx:76]: When Input is replaced with TextInput onChange event type definition will not be required, remove it

File: docs/src/components/MdxPage/MdxPageTabs/PropsTable/PropsTable.tsx [hunk 3]
```diff
@@ -3,11 +3,21 @@ import { Table, Text, useMantineTheme } from '@mantine/core';
...
+  component: string;
+  searchValue: string;
+}) {
   const theme = useMantineTheme();
 
   const rows = Object.keys(PROPS_DATA[component].props).map((propKey) => {
     const prop = PROPS_DATA[component].props[propKey];
+
+    if (searchValue && !prop.name.includes(searchValue)) {
+      return <React.Fragment key={propKey} />;
```
Comment (rtivital) [inline docs/src/components/MdxPage/MdxPageTabs/PropsTable/PropsTable.tsx:19]: Just return null

File: docs/src/components/MdxPage/MdxPageTabs/PropsTable/PropsTable.tsx [hunk 4]
```diff
@@ -3,11 +3,21 @@ import { Table, Text, useMantineTheme } from '@mantine/core';
...
 import PROPS_DATA from '../../../../../.docgen/docgen.json';
 
-export default function PropsTable({ component }: { component: string }) {
+export default function PropsTable({
+  component,
+  searchValue,
+}: {
+  component: string;
+  searchValue: string;
+}) {
```
Comment (rtivital) [inline docs/src/components/MdxPage/MdxPageTabs/PropsTable/PropsTable.tsx:12]: Put type in separate interface:

```tsx
interface PropsTableProps {
  component: string;
  searchValue: string;
}
```

File: docs/src/components/MdxPage/MdxPageTabs/PropsTable/PropsTable.tsx [hunk 5]
```diff
@@ -3,11 +3,21 @@ import { Table, Text, useMantineTheme } from '@mantine/core';
 import DocsSection from '../../../DocsSection/DocsSection';
 import PROPS_DATA from '../../../../../.docgen/docgen.json';
 
-export default function PropsTable({ component }: { component: string }) {
+export default function PropsTable({
+  component,
+  searchValue,
```
Comment (rtivital) [inline docs/src/components/MdxPage/MdxPageTabs/PropsTable/PropsTable.tsx:8]: rename searchValue -> query

File: docs/src/components/MdxPage/MdxPageTabs/MdxPageTabs.tsx [hunk 6]
```diff
@@ -65,6 +66,17 @@ export function MdxPageTabs({ body, frontmatter, headings }: MdxPage) {
               style={{ maxWidth: 1080, margin: 'auto', marginTop: 24 }}
               className={classes.tabContent}
             >
+              <Input
+                variant="default"
```
Comment (rtivital) [inline docs/src/components/MdxPage/MdxPageTabs/MdxPageTabs.tsx:70]: Remove variant, it sets dynamically based on color scheme

**Review by rtivital — CHANGES_REQUESTED** [review]
Thanks for a new PR, please fix these changes so that it can be merged

**Review by ghana7989 — COMMENTED** [review]
I fixed them

**Comment by ghana7989** [comment]
Is there anything else you want me to update?

**Review by rtivital — APPROVED** [review]

**Comment by rtivital** [comment]
Thanks for PR, everything works good

**Pull request merged**